docker-compose.yml

```diff
@@ -357,6 +368,7 @@ services:
       - MONGO_HOST=mongo_arm64
       - SERVICESTACK_REDIS_HOST=servicestackredis_arm64:6379
       - STACKEXCHANGE_REDIS_HOST=stackexchangeredis_arm64:6379
+      - ELASTICSEARCH7_HOST=elasticsearch7_arm64:9200
```

Comment:
You'll need to create the `elasticsearch7_arm64` service too (and add it to StartDependencies.ARM64)

Reply:
You mean this one? 😄  https://github.com/DataDog/dd-trace-dotnet/blob/master/docker-compose.yml#L9

Reply:
Err.... yeah... exactly like that one. Silly me, thinking we would be using elasticsearch 5 for elasticsearch5 tests 😬